bin/dev

```diff
@@ -72,6 +73,16 @@ HEREDOC
     args.unshift("docker-compose", "-f /usr/local/dev-env/docker/dev-env.yml", "-p devenv")
     system(args.join(" "))
   end
+
+  def cmd(args)
+    args = ["/bin/bash", "-l"] if args.empty?
+    container_id = `docker container ls -q -f name=ermanagement_web_`.strip
+    if container_id == ""
+      puts "Container is not running"
+    else
+      system("docker exec -i -t #{container_id} #{args.join(" ")}")
+    end
```

Comment:
We can convert this to work more like `dev run worker bash` if no args are present, or `dev run worker {something}` if args are present.